client/src/components/table/component.tsx

```diff
@@ -129,10 +129,12 @@ const Table: React.FC<TableProps> = ({
     [handleApiSorting, sortingMode, updateTableProps],
   );
 
+  // TO-DO: This is a hack to make sure the table is always rendered.
```

Comment:
Is this to synchronise the internal table store with our data? Whenever it changes?

Then it's not a hack, I do it too to to sync the page number/size with the API requests, it's the cleanest way I found 💀 

Maybe I should add some comments in my branch 😂

Reply:
feel free to remove the comment, as you make changes here